src/qt/paymentrequestplus.cpp

```diff
@@ -125,7 +119,7 @@ bool PaymentRequestPlus::getMerchant(X509_STORE* certStore, QString& merchant) c
     // The first cert is the signing cert, the rest are untrusted certs that chain
     // to a valid root authority. OpenSSL needs them separately.
     STACK_OF(X509) *chain = sk_X509_new_null();
-    for (int i = certs.size()-1; i > 0; i--) {
+    for (size_t i = certs.size() - 1; i > 0; i--) {
```

Comment:
NACK on changing this. There is check for certs.empty() that prevents this from becoming a bug, but for loops that count down, I generally prefer to use signed types. In this case, `i` will never visit 0, but normally a loop like this would look like:

    for (int i = certs.size() - 1; i >= 0; i--) {

If you then change the int to e.g. size_t, it will loop forever.